### DISCUSSION TITLE
How to enable vue reactivity transform experimental feature?

### DISCUSSION THREAD

**Discussion by aislanmaia** [quasarframework/quasar#12450] [General - Components / Directives / etc] [answered]
Looking for to use one of the most cool future features of Vue 3.3, the reacitvity transform.

With it we can use "props destructuring" and more.

See it here: https://github.com/vuejs/rfcs/discussions/369 && https://vuejs.org/guide/extras/reactivity-transform.html

**Comment by rstoenescu** [comment, marked as answer]
quasar.conf.js > build > vueLoaderOptions: { reactivityTransform: true }

**Reply by aislanmaia** [reply]
Thanks!!! Unfortunately using $ref doesnt get recognized in the script.

**Reply by mv-go** [reply]
try this `quasar.config.js -> build.viteVuePluginOptions.reactivityTransform -> true`. This way it will start build and working just fine. Linter still thinks these are undef, just gotta figure this one out now
